bsp/esp32_s3_eye/include/bsp/config.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
```

Comment:
Wouldn't be better to move this file to private?

Reply:
We include this file from public `esp32_s3_eye.h` so it must be public